torchgeo/trainers/segmentation.py

```diff
@@ -29,10 +29,21 @@
 
 
 class SemanticSegmentationTask(pl.LightningModule):
-    """LightningModule for semantic segmentation of images."""
+    """LightningModule for semantic segmentation of images.
+
+    Supports `Segmentation Models Pytorch
+    <https://github.com/qubvel/segmentation_models.pytorch>`_
+    as an architecture choice in combination with any of these
+    `TIMM encoders <https://smp.readthedocs.io/en/latest/encoders_timm.html>`_.
+    """
 
     def config_task(self) -> None:
-        """Configures the task based on kwargs parameters passed to the constructor."""
+        """Configures the task based on kwargs parameters passed to the constructor.
+
+        .. versionchanged:: 0.3.2
+            Improve error message of supported segmentation architectures
+            and loss function.
```

Comment:
Same here. versionchanged/added should only ever be 2 digits because anything worth documenting is either a new feature or a change in backwards compatibility.

Reply:
ah okay, I wasn't aware of the exact definition, I thought since I changed the code within the function, I should track it with versionchanged

Reply:
Nah, only API or behavior changes